File: ios/ImageProcessingSDK.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(ImageProcessingSDK, NSObject)

RCT_EXTERN_METHOD(isImageBlurred:(NSString *)imageUrl resolve:(RCTPromiseResolveBlock)resolve reject:(RCTPromiseRejectBlock)reject)

@end

File: ios/ImageProcessingSDK.swift
@objc(ImageProcessingSDK)
class ImageProcessingSDK: NSObject {

  @objc
  func isImageBlurred(_ imageUrl: String, resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) {
    resolve(true)
  }
}
